src/Surfnet/StepupMiddleware/CommandHandlingBundle/Identity/CommandHandler/IdentityCommandHandler.php

```diff
@@ -278,4 +300,20 @@ private function assertIsValidLocale(Locale $locale)
             );
         }
     }
+
+    private function assertSecondFactorIsAllowedFor(SecondFactorType $secondFactor, Institution $institution)
+    {
+        $isSecondFactorAllowed = $this->allowedSecondFactorListService->isSecondFactorAllowedFor(
+            $secondFactor,
+            new ConfigurationInstitution($institution->getInstitution())
+        );
+
+        if ($isSecondFactorAllowed === false) {
```

Comment:
With not "hiding" the call, this would become `if (!$allowedSecondFactorList->allows($secondFactorType)) {` which to me seems a better indicator...